web/styles/dark_theme.css

```diff
@@ -649,7 +657,7 @@
     .tippy-box[data-theme~="light-border"],
     .dropdown-menu ul,
     .dropdown .dropdown-menu {
-        background-color: hsl(212deg 32% 14%);
+        background-color: var(--color-background-popover-light-theme);
```

Comment:
What's the story behind this name? Do you mean `color-background-popover-light-border`? I think we can leave off the `light-border` part and just call it `popover`; `light-border` is just the Tippy name for the theme.

I also think if we're defining CSS variables, we should be able to just use those variables in the main definition and delete these `dark_theme.css` override blocks.

Reply:
Fixed!